app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is a class attribute, which means it is shared across all instances of `Person`. This could lead to unexpected behavior if multiple instances are created with the same name. Consider using instance attributes or a different approach to manage relationships.